maquetado/login.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Log In</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+    <form class="info-container column">
+      <img src ="./assets/wolox.png" class="login-img" alt="Wolox">
+      <label for="email" class="input-description">Email</label>
+      <input name="email" id="email" class="input-field" type="email">
+      <label for="password" class="input-description">Password</label>
+      <input name="password" id="password" class="input-field" type="password">
+      <button href="tbdef" class="login-detail">Log in</button>
+      <button href="tbdef" class="sign-up-detail">Sign Up</button>
```

Comment:
This should be an `a` tag instead. Since this button is supposed to take you to another page. Also, you should use `href="#"` as placeholder when you don't have an address yet.